test/simple_select.test.js

```diff
@@ -0,0 +1,160 @@
+/* eslint no-shadow:[0] */
+import test from 'tape';
+import mapboxgl from 'mapbox-gl-js-mock';
+import GLDraw from '../';
+import { click, accessToken, createMap, features } from './test_utils';
+import spy from 'sinon/lib/sinon/spy'; // avoid babel-register-related error by importing only spy
+import stub from 'sinon/lib/sinon/stub'; // avoid babel-register-related error by importing only stub
+
+mapboxgl.accessToken = accessToken;
+
+const EVENT_TIME = 32;
+
+var feature = features.point;
+
+var map = createMap();
+spy(map, 'fire');
+
+var Draw = GLDraw();
+map.addControl(Draw);
+
+var afterNextRender = function(cb) {
+  setTimeout(cb, 32);
+}
+
+var cleanUp = function(cb) {
+  Draw.deleteAll();
+  map.fire.reset();
+  afterNextRender(cb);
+}
+
+var getFireArgs = function() {
+  var args = [];
+  for (var i=0; i<map.fire.callCount; i++) {
+    args.push(map.fire.getCall(i).args);
+  }
+  return args;
+}
+
+var makeMouseEvent = function(lng, lat, shiftKey = false) {
```

Comment:
It's be great to extract this out into a `test_util`. We'll need it in other modes.